test/Datadog.Trace.ClrProfiler.IntegrationTests/HttpClientTests.cs

```
@@ -85,7 +85,7 @@ public void HttpClient_TracingDisabled()
             }
         }
 
-        [Fact]
+        [Fact(Skip = "Until WebClient instrumentation is available")]
```

This comment makes it sounds like this PR is removing existing instrumentation for `WebClient` without a replacement. Is this correct?

---

No, just that there were some existing tests, but I didn't remove any instrumentation for it.

---

Then why are we now skipping the test that we used to run before?

---

Because there is still no instrumentation that covers `WebClient`. That is one of the classes of instrumentation that we lost because of the earlier changes. I haven't been able to successfully instrument it so I'm not including it in this PR.

---

To clarify, we're skipping this test because adding `LoaderOptimzation.MultiDomainHost` to the sample app makes the `WebClient` integration fail. Resolving this comment.

---

That may be true, I don't know for sure. But removing the loader optimization, the test still fails.

---

If the test was not failing before and it's failing now, shouldn't we try to figure out why it's failing instead of skipping it? A failing test reason enough to block merging this PR. (Unresolving)

---

Ok, I agree.